sample-node-config/aws/README.md

```diff
@@ -0,0 +1,234 @@
+
+
+- you have terraform and aws installed on your system.
+- you have built the required 2 folder structures under aws folder:
+    + credentials.
+    + env.
+
+## Project structure to build
+```
+.
++-- credentials
+|   +-- arnaud-cardano.vk
+|   +-- arnaud-hydra.vk
+|   +-- cardano-key.sk
+|   +-- cardano-key.vk
+|   +-- cardano.addr
+|   +-- hydra-key.sk
+|   +-- hydra-key.vk
++-- env
+|   +-- personal.pem
+```
+
+### Building ***credentials***
+This should only be done once, when starting afresh.
+This is a folder you must create yourself to store all blockchain related credential files.
+
+1. ask to your party members for their verification keys (for this template, the party member is ***arnaud***)
+
+2. generate your cardano signing and verification keys.
+at the root of `hydra-poc` project, execute:
+    ```sh
+    $ cardano-cli address key-gen --signing-key-file cardano-key.sk --verification-key-file cardano-key.vk
+    ```
+    > `cardano-key.sk` and `cardano-key.vk` will be generated at the root of the `hydra-poc` project.
+
+3. generate your hydra signing and verification keys.
+at the root of `hydra-poc` project, execute:
+    ```sh
+    $ cabal run hydra-tools gen-hydra-key
+    ```
+    > `hydra-key.sk` and `hydra-key.vk` will be generated at the root of the `hydra-poc` project.
+
+4. generate your cardano address
+at the root of `hydra-poc` project, execute:
+    ```sh
+    $ cardano-cli address build --payment-verification-key-file ./cardano-key.vk --testnet-magic 1 > cardano.addr
+    ```
+    > `cardano.addr` will be generated at the root of the `hydra-poc` project.
+
+5. move under `credentials` folder:
+    - the party members verification keys: `arnaud-cardano.vk` and `arnaud-hydra.vk`.
+    - your personal hydra keys: `hydra-key.sk` and `hydra-key.vk`.
+    - your personal cardano keys: `cardano-key.sk` and `cardano-key.vk`.
+    - your personal cardano address: `cardano.addr`.
+
+    ```sh
+    $ mv *.vk sample-node-config/aws/credentials/
+    $ mv *.sk sample-node-config/aws/credentials/
+    $ mv cardano.addr sample-node-config/aws/credentials/
+    ```
+
+### Building ***env***
+This should only be done once, when starting afresh.
+This is a folder you must create yourself to store your personal aws credentials.
+
+go to ***EC2 Dashboard*** > ***Network & Security*** > ***Key Pairs*** and create one (for this template, the key pair is ***personal*** as mentioned).
+Then move it under `env` folder.
+
+> Default values are good enough (type: RSA & OpenSSH format: *.pem).
+
+> Creating a new pair will automatically generate a file and download it (one time action).
+
+## Initialise Terraform
+This should only be done once, when starting afresh.
+At the root of `aws` project execute:
+```sh
+$ terraform init
+```
+
+### Alter terraform variables
+
+Alter the file variables.tf found in `sample-node-config/aws/` to reflect your key name, github account etc.
```

Comment:
I do not find any explanation for the ghaccount variable in variables.tf

Reply:
OK, I found some text mentionnant GitHub account so I guess this is what this variable is. I'm wondering why I would need a GitHub account to run this but we'll see :)

Reply:
jaja i love your comments!
your github account is used on configure-testnet script to pull and import your github gpg keys.

Reply:
I've removed this part as... well it seems it can work without